packages/client/src/views/RecordAudit/ActionDetailsModal.tsx

```diff
@@ -576,14 +576,18 @@ export const ActionDetailsModal = ({
           <>{userName}</>
           <span> — {getFormattedDate(actionDetailsData.date)}</span>
         </div>
-        <ActionDetailsModalListTable
-          actionDetailsData={actionDetailsData}
-          actionDetailsIndex={actionDetailsIndex}
-          registerForm={registerForm}
-          intl={intl}
-          offlineData={offlineData}
-          draft={draft}
-        />
+        {!actionDetailsData.action ||
+          (actionDetailsData.action &&
+            actionDetailsData.action === RegAction.RequestedCorrection && (
+              <ActionDetailsModalListTable
+                actionDetailsData={actionDetailsData}
+                actionDetailsIndex={actionDetailsIndex}
+                registerForm={registerForm}
+                intl={intl}
+                offlineData={offlineData}
+                draft={draft}
+              />
+            ))}
```

Comment:
@sadmananik We need to figure out a different way to do this as #4286 won't work with this change and some new ones might come up in the future as well